modules/ding_ddbasic/modules/ding_ddbasic_opening_hours/plugins/content_types/weekly_opening_hours.inc

```diff
@@ -29,6 +29,10 @@ function ding_ddbasic_opening_hours_weekly_opening_hours_content_type_render($su
     $node = node_load($conf['node']);
   }
 
+  if (empty(opening_hours_present_on_node($node->nid)) && variable_get('opening_hours_hide_on_empty_ding_library', 0) == 1) {
+    return $block;
+  }
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-1142

Reply:
This seems right. However is `opening_hours_hide_on_empty_ding_library` set properly?
In culture I see this PR that is changed with this case: https://github.com/easySuite/culture/pull/38/files.

Reply:
This should hide the opening hours block on Library CT node. This is good functionality.
Make sure that other changes from mentioned PR is also moved prior to testing.

Reply:
https://github.com/easySuite/ding2/pull/664/files#diff-e5d586c7c44178bdfa9b2995c757f6f5512e9e87c7169c85205a8ae21b7ada36R51